Wrappers/Python/ccpi/web_viewer/trame_viewer.py

```diff
@@ -174,25 +182,25 @@ def create_orientation_radio_buttons(self, disabled: bool = False):
         )
 
     def construct_slice_window_slider(self, disabled: bool = False):
-        if self.cmax > 100:
+        if not self.window_level_sliders_are_percentages:
             # Use actual value
             min_value = self.cmin
             max_value = self.cmax
             step = 1
```

Comment:
This is supposing that the data are integers or anyway > 1. In normalised datasets what do we do? Force percents? Is it available?

I'd actually suggest to decide to have, say 1000 positions of the slider, and then calculate the `step` accordingly.

Reply:
I do not like this idea, it would be quite hard to properly select specific attenuation values where the dataset allows it.

Reply:
This PR is leaving the decisison for whether to have % or not, and slider step how it was originally, I did not intend to tackle that here, and we should address in a separate PR

Reply:
I do not think a slider that is between 0 and 100 with 1000 steps between, for all data makes much sense.